File: service/php-symfony/src/Entity/Account.php
<?php

namespace App\Entity;

use App\Repository\AccountRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Uid\UuidV4;

#[ORM\Entity(repositoryClass: AccountRepository::class)]
class Account
{
    #[ORM\Id]
    #[ORM\Column(type: 'uuid', unique: true)]
    #[ORM\GeneratedValue(strategy: "CUSTOM")]
    #[ORM\CustomIdGenerator(class: 'doctrine.uuid_generator')]
    private $id;

    #[ORM\Column(type: 'text', nullable: true)]
    private $login;

    #[ORM\Column(type: 'datetimetz', nullable: true)]
    private $creation_date;

    #[ORM\OneToMany(mappedBy: 'account', targetEntity: ListEntity::class)]
    private $lists;

    public function __construct()
    {
        $this->lists = new ArrayCollection();
    }

    public function getId(): ?UuidV4
    {
        return $this->id;
    }

    public function getLogin(): ?string
    {
        return $this->login;
    }

    public function setLogin(?string $login): self
    {
        $this->login = $login;

        return $this;
    }

    public function getCreationDate(): ?\DateTimeInterface
    {
        return $this->creation_date;
    }

    public function setCreationDate(?\DateTimeInterface $creation_date): self
    {
        $this->creation_date = $creation_date;

        return $this;
    }

    /**
     * @return Collection<int, ListEntity>
     */
    public function getLists(): Collection
    {
        return $this->lists;
    }

    public function addList(ListEntity $list): self
    {
        if (!$this->lists->contains($list)) {
            $this->lists[] = $list;
            $list->setAccount($this);
        }

        return $this;
    }

    public function removeList(ListEntity $list): self
    {
        if ($this->lists->removeElement($list)) {
            // set the owning side to null (unless already changed)
            if ($list->getAccount() === $this) {
                $list->setAccount(null);
            }
        }

        return $this;
    }
}

File: service/jvm-quarkus/src/main/java/com/ecobenchmark/entities/Account.java
package com.ecobenchmark.entities;

import org.hibernate.annotations.GenericGenerator;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import java.time.LocalDateTime;
import java.util.UUID;

@Entity
public class Account extends PanacheEntity {

    @Id
    @GeneratedValue(generator = "UUID")
    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
    private UUID id;

    private String login;

    private LocalDateTime creationDate;

    @OneToMany(mappedBy = "account")
    private ListEntity lists;

    public Account(String login, LocalDateTime creationDate, ListEntity lists) {
        this.login = login;
        this.creationDate = creationDate;
        this.lists = lists;
    }

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public LocalDateTime getCreationDate() {
        return creationDate;
    }

    public void setCreationDate(LocalDateTime creationDate) {
        this.creationDate = creationDate;
    }

    public ListEntity getLists() {
        return lists;
    }

    public void setLists(ListEntity lists) {
        this.lists = lists;
    }
}
